### ISSUE TITLE
Respond to _ah/start?

### ISSUE THREAD

**Issue by achuinard** [renaudcerrato/appengine-maven-repository#23]
I deployed this awesome repo to a new App Engine app yesterday and it worked for the first time. Then, this morning, the app stopped working, and I saw this in the logs:

```
2019-11-13 17:34:02.397 CST
The request failed because the instance could not start successfully
2019-11-13 17:34:02.397 CST
Process terminated because it failed to respond to the start request with an HTTP status code of 200-299 or 404.
```

I don't understand why it worked on deploy then stopped. It could have something to do with the instance scaling setup I added to `appengine-web.xml`.

```
<basic-scaling>
        <max-instances>1</max-instances>
        <idle-timeout>3m</idle-timeout>
    </basic-scaling>
```

It scales the app down to 0 instances when not in use.

I added an `_ah/start` controller and will make a PR...I just don't get why 1) the app worked and 2) no one has dealt with this before? Especially because it was reproducible locally just using `./gradlew appengineRun`.

**Comment by achuinard** [comment]
PR in #24.

**Comment by renaudcerrato** [comment]
Mmmmm... It looks like manual and basic scaling requires an handler to `/_ah/start` :

https://cloud.google.com/appengine/docs/standard/java/how-instances-are-managed#startup

**Comment by achuinard** [comment]
Nice catch in the docs, I did not see that. I was running into OutOfMemory issues uploading a ~26MB JAR, which was 6MB under the 32MB GAE threshold, but upgrading my instance to something with more memory fixed it. However, I only use my repo when I'm working on side projects, so I don't need it very often. Basic / manual scaling let me drop the instance count to 0, which I don't know automatic scaling does - but then again I might have missed that in the docs too 😆

**Comment by renaudcerrato** [comment]
Closed by #24
